editor/editor_tests/xml_feature_test.cpp

```diff
@@ -499,18 +499,6 @@ UNIT_TEST(XMLFeature_SocialContactsProcessing)
     osm::EditableMapObject emo;
     editor::FromXML(xmlFeature, emo);
 
-    // Read and write "contact:facebook" to apply normalization.
```

Comment:
Why these tests are not needed?

Reply:
Originally I wrote this test with steps:

1. Convert XML into EditableMapObject
2. Normalize social contacts to get shorter values
3. Convert EditableMapObject to XML
4. Verify XML content.

But after this PR step 2 is not needed.